include/tvm/relay/attrs/nn.h

```diff
@@ -596,11 +596,13 @@ struct Conv2DTransposeAttrs : public tvm::AttrsNode<Conv2DTransposeAttrs> {
 /*! \brief Attributes used in dilate operator */
 struct DilateAttrs : public tvm::AttrsNode<DilateAttrs> {
   Array<IndexExpr> strides;
+  double dilation_value;
```

Comment:
Why double vs float?

Reply:
This is parallel to `pad_value` of `PadAttrs` which is `double`.